packages/jsii-dotnet-runtime/src/Amazon.JSII.Runtime/Services/Runtime.cs

```diff
@@ -9,11 +10,22 @@ public class Runtime : IRuntime
         public Runtime(INodeProcess nodeProcess)
         {
             _nodeProcess = nodeProcess ?? throw new ArgumentNullException(nameof(nodeProcess));
+            if (Environment.GetEnvironmentVariable("JSII_DEBUG") != null)
+            {
+                Task.Run(() => RedirectStandardError());
```

Comment:
Uses same logic as Java. If there is an error, we log to error. This is if we expect there to be informational messages constantly logged to error, in the case of JSII_DEBUG being set.

Reply:
Not sure I understand the response

Reply:
If JSII_DEBUG is not set, then we don't echo the contents of JSII STDERR to .NET STDERR constantly. We expect the contents of STDERR to only contain error massages, so we only read the stream when there is an exception thrown or some other extraordinary condition.

If it is set, we want echo these contents in real time, since they are informational in nature.

This logic was copied from the java runtime code.

Reply:
Got it. thanks for the details.